.gitignore

```diff
@@ -9,3 +9,5 @@ Cargo.lock
 # These are backup files generated by rustfmt
 **/*.rs.bk
 
+# IDE
+.idea
```

Comment:
As a fellow IntelliJ user, I'd recommend  defining a `[core.excludesfile]` in your `~/.gitconfig` so the `.gitignore` doesn't have to incorporate errata for every editor/IDE's metadata/temp files.